src/main/java/kg/jarkyn/server/ReceivingSocket.java

```diff
@@ -0,0 +1,22 @@
+package kg.jarkyn.server;
+
+import kg.jarkyn.server.Incoming.Requester;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public class ReceivingSocket {
```

Comment:
because of conflict with java.net.ServerSocket, I used to have it as ServerSideSocket to distinct, but then it is just buttery butter, so went with ReceivingSocket, I actually like this one, for now will stick with it